src/Tasks/Common/Resources/xlf/Strings.cs.xlf

```diff
@@ -449,9 +449,14 @@
         <target state="translated">NETSDK1102: Pro vybranou konfiguraci publikování se optimalizace velikosti sestavení nepodporuje. Ujistěte se, že publikujete samostatnou aplikaci.</target>
         <note>{StrBegin="NETSDK1102: "}</note>
       </trans-unit>
-      <trans-unit id="ILLink_Info">
-        <source>Optimizing assemblies for size, which may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink</source>
-        <target state="translated">Optimalizuje se velikost sestavení, což může změnit chování aplikace. Nezapomeňte po vydání provést testy. Další informace: https://aka.ms/dotnet-illink</target>
+      <trans-unit id="ILLinkOptimizedAssemblies">
+        <source>Assemblies were optimized for size. See: https://aka.ms/dotnet-illink</source>
```

Comment:
Unrelated: The Czech translation is ... oh god... I always wondered why they have to translate terms like "Assembly" (which in this case is translated verbatim... like if you're assembling a LEGO set).